apps/passport-server/src/services/telegramService.ts

```diff
@@ -35,6 +40,7 @@ export class TelegramService {
   private context: ApplicationContext;
   private bot: Bot;
   private rollbarService: RollbarService | null;
+  private IS_LOCAL_SERVER: boolean;
```

Comment:
Since `process.env.PASSPORT_SERVER_URL` is consistent for the global scope, this should just be a file-level constant, not a class variable

Reply:
@rrrliu I was getting an error when it was a file level constant for some reason because the values were undefined on initial load.

Reply:
Got it, in that case I prefer having a function like `isLocalServer()` in a separate utils file. Let me know if it makes sense why that's preferable over a class field. For now, let's just add a note to fix this (it's low-priority).

Reply:
Cool. I can update in a subsequent PR.

Reply:
https://github.com/proofcarryingdata/zupass/issues/658

Reply:
Fixed e7a86f5a0a59707442a676900dc0f68a12154a32